doc/man3/SSL_alert_type_string.pod

```diff
@@ -69,7 +69,7 @@ SSL_alert_desc_string_long():
 
 =over 4
 
-=item "CN"/"close notify"
+=item "CN"/close_notify
```

Comment:
This instance here is an exception and needs to be reverted. It is a string return value documented in the  `RETURN VALUES` section.